quakemigrate/plot/event.py

```diff
@@ -148,19 +148,21 @@ def _plot_waveform_gather(ax, lut, event, idx):
     """
 
     # --- Predicted traveltimes ---
-    ttpf, ttsf = [lut.traveltime_to(phase, idx) for phase in "PS"]
-    ttp = [(event.otime + tt).datetime for tt in ttpf]
-    tts = [(event.otime + tt).datetime for tt in ttsf]
-    range_order = abs(np.argsort(np.argsort(ttp)) - len(ttp)) * 2
+    traveltimes = [lut.traveltime_to(phase, idx) for phase in lut.phases]
+    arrivals = [[(event.otime + tt).datetime for tt in tt_f]
+                for tt_f in traveltimes]
+
+    range_order = abs(np.argsort(np.argsort(arrivals[0]))
+                      - len(arrivals[0])) * 2
     s = (ax.get_window_extent().height / (max(range_order)+1) * 1.2) ** 2
-    max_tts = max(ttsf)
-    for tt, c, phase in zip([ttp, tts], PICK_COLOURS, "PS"):
-        ax.scatter(tt, range_order, s=s, c=c, marker="|", zorder=5, lw=1.5,
-                   label=f"Modelled {phase}")
+
+    for arrival, c, phase in zip(arrivals, PICK_COLOURS, lut.phases):
+        ax.scatter(arrival, range_order, s=s, c=c, marker="|", zorder=5,
+                   lw=1.5, label=f"Modelled {phase}")
 
     # --- Waveforms ---
     times_utc = event.data.times(type="UTCDateTime")
-    mint, maxt = event.otime - 0.1, event.otime + max_tts*1.5
+    mint, maxt = event.otime - 0.1, event.otime + max(traveltimes[-1])*1.5
```

Comment:
Is it necessary to use `traveltimes[-1]`? (I assume this corresponds to the last-added phase / last phase in `lut.phases`, which may not always be the slowest). E.g.:
```    mint, maxt = event.otime - 0.1, event.otime + max([max(tts) for tts in traveltimes])*1.5```

Reply:
It does - yeah, might as well handle in this instance as it is straightforward. I have a feeling there are a few instances where a similar order assumption is used elsewhere in the code, so would need a proper review to make sure.

Reply:
Addressed in 1957c29.